### ISSUE TITLE
LaTeX: simple quotes not preserved in literalinclude

### ISSUE THREAD

**Issue by shimizukawa** [sphinx-doc/sphinx#636]
The "includeliteral" directive does not preserve single quotes in LaTeX ouput but convert them in curved single quotes.

For instance:

1) Source:

```
.. literalinclude:: quotes.txt
```

2) Included file (quotes.txt):

```
print 'bla'
```

3) Displayed output:

```
print ’bla’
```

4) Generated LaTeX source:

```
\begin{Verbatim}[commandchars=\\\{\}]
\PYG{k}{print} \PYG{l+s}{'}\PYG{l+s}{bla}\PYG{l+s}{'}                                  
\end{Verbatim}
```

---
- Bitbucket: https://bitbucket.org/birkenfeld/sphinx/issue/636
- Originally reported by: Anonymous
- Originally created at: 2011-02-15T10:41:13.258

**Comment by shimizukawa** [comment]
_From Georg Brandl on 2011-09-21 17:22:57+00:00_

This is simply how these quotes look in the font used.

**Comment by shimizukawa** [comment]
_From Georg Brandl on 2013-03-31 13:57:33+00:00_

Removing version: 0.6.6 (automated comment)

**Comment by shimizukawa** [comment]
_From Georg Brandl on 2014-03-25 09:25:43+00:00_

Actually no :(

**Comment by shimizukawa** [comment]
_From Georg Brandl on 2014-03-25 09:27:47+00:00_

Closes #636: Keep straight single quotes in literal blocks in the LaTeX build.

→ <<cset 53abebda699551a460bb0b8e60982304e5b96415>>
